prometheus/pkg/adapter/adapter.go

```diff
@@ -159,9 +157,8 @@ func (a *prometheusAdapter) makeEvent(payload interface{}) (*cloudevents.Event,
 	event.SetSource(a.source)
 	event.SetID(string(uuid.NewUUID()))
 	event.SetType(v1alpha1.PromQLPrometheusSourceEventType)
-	event.SetDataContentType(cloudevents.ApplicationJSON)
 
-	if err := event.SetData(payload); err != nil {
+	if err := event.SetData(cloudevents.ApplicationJSON, payload); err != nil {
 		return nil, err
```

Comment:
Can you wrap the error here specifying that setting the data field failed?

Reply:
Like https://github.com/knative/eventing-contrib/pull/1202/files#diff-68db06af632e5aef92f2b1e8d54e3567R181

Reply:
ok

Reply:
done